uui/components/datePickers/DatePickerBody.module.scss

```diff
@@ -2,9 +2,9 @@
 
 .root {
     --uui-dtp_body-title-bg: var(--uui-control-bg);
-    --uui-dtp_body-bg-selected: var(--uui-primary-50);
+    --uui-dtp_body-cell-bg-selected: var(--uui-primary-50);
```

Comment:
what do you mean by 'cell'? May be 'day'?

Reply:
It cell for matrix of days, months, years